class-00/lab-a/lab/js/app.js

```diff
@@ -61,7 +61,7 @@ function handleClick(event) {
   for(let i = 0; i < names.length; i++){
     if(event.target.id === allProducts[i].name) {
       allProducts[i].votes += 1;
-      console.log(event.target.id + ` has ` + allProducts[i].votes + ` votes in ` + allProducts[i].views + ` views`);
+      console.log(event.target.id + ` has + ${allProducts[i].votes} + votes in + ${allProducts[i].views} + views`);
```

Comment:
You don't need to concatenate with `+` when using template literal notation!

`${event.target.id} has ${allProducts[i].votes} votes in ${allProducts[i].views} views`